client/src/assets/images/icons/index.js

```diff
@@ -473,3 +473,37 @@ export const ErrorIcon = () => (
     />
   </svg>
 );
+
+export const AppIcon = () => (
+  <svg
+    aria-hidden="true"
```

Comment:
Consider extracting this as:
```javascript
const Icon = props => (
  <svg
    className="svg-inline--fa fa-coffee fa-w-20"
    data-prefix="fas"
    role="img"
    viewBox="0 0 640 512"
    xmlns="http://www.w3.org/2000/svg"
    {..props}
  />
);
```

Then you can use it like this:

```javascript
export const AppIcon = () => (
  <Icon data-icon="coffee">
    <path
      d="M192 384h192c53 0 96-43 96-96h32c70.6 0 128-57.4 128-128S582.6 32 512 32H120c-13.3 0-24 10.7-24 24v232c0 53 43 96 96 96zM512 96c35.3 0 64 28.7 64 64s-28.7 64-64 64h-32V96h32zm47.7 384H48.3c-47.6 0-61-64-36-64h583.3c25 0 11.8 64-35.9 64z"
      fill="#ef9b1f"
    />
  </Icon>
)
```

This reduces duplication a lot and makes all the icons use consistent sizes and parameters. IMHO, `aria-hidden="true"` should be deleted as this is a false statement and may confuse accessibility tools - please see https://css-tricks.com/accessible-svgs/#article-header-id-8